sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -142,6 +166,10 @@ private Crashes() {
         mCrashesListener = DEFAULT_ERROR_REPORTING_LISTENER;
         mUnprocessedErrorReports = new LinkedHashMap<>();
         mErrorReportCache = new LinkedHashMap<>();
+        HandlerThread thread = new HandlerThread(THREAD_NAME);
```

Comment:
Why using handler thread for one shot tasks?
This case is legit for ASyncTask as after we load what we need, there won't be any other use of the thread until application dies.